framework/wazuh/core/tests/test_manager.py

```diff
@@ -116,13 +116,14 @@ def test_get_ossec_logs(log_format):
     logs = get_logs(json_log=log_format == LoggingFormat.json).splitlines()
 
     with patch("wazuh.core.manager.get_wazuh_active_logging_format", return_value=log_format):
-        with pytest.raises(WazuhInternalError, match=".*1020.*"):
-            get_ossec_logs()
-
-        with patch('wazuh.core.manager.exists', return_value=True):
-            with patch('wazuh.core.manager.tail', return_value=logs):
-                result = get_ossec_logs()
-                assert all(key in log for key in ('timestamp', 'tag', 'level', 'description') for log in result)
+        if log_format == LoggingFormat.plain:
+            with pytest.raises(WazuhInternalError, match=".*1020.*"):
+                get_ossec_logs()
+        else:
+            with patch('wazuh.core.manager.exists', return_value=True):
+                with patch('wazuh.core.manager.tail', return_value=logs):
+                    result = get_ossec_logs()
+                    assert all(key in log for key in ('timestamp', 'tag', 'level', 'description') for log in result)
```

Comment:
Hi Gaston, what is the test that failed? I can't find the one that failed

Reply:
Hello @javiersanchz, there is no test failing but we are skipping some validations with the _if-else_ condition. I meant that if the tests fail when it's removed, it may be something else causing issues.